google/resource_container_cluster_test.go

```diff
@@ -645,6 +675,18 @@ resource "google_container_cluster" "primary" {
 	}
 }`, acctest.RandString(10))
 
+var testAccContainerCluster_withAddons = fmt.Sprintf(`
+resource "google_container_cluster" "primary" {
+	name = "cluster-test-%s"
+	zone = "us-central1-a"
+	initial_node_count = 3
+
+	addons_config {
+        http_load_balancing { disabled = true }
```

Comment:
can you fix the formatting on this?

Reply:
Argh, tabs/spaces in strings are constantly vexing me!

Fixed